src/create_torrent.cpp

```diff
@@ -643,12 +643,12 @@ namespace {
 				if (url.second != current_tier)
 				{
 					current_tier = url.second;
-					trackers.push_back(std::move(tier));
+					trackers.emplace_back(std::move(tier));
```

Comment:
Could you change this in the next suitable update (2.1 or 3.0) so that the trackers are in the same array as some structures (even as `announce_entry`), and not different arrays for trackers and tiers. In addition, you would then be able to store the original sources of trackers in the resume data, which may be useful to someone.